jax/_src/pjit.py

```diff
@@ -493,6 +493,16 @@ def common_infer_params(pjit_info_args, *args, **kwargs):
       in_type, dbg, device_or_backend_set, HashableFunction(out_tree, closure=()),
       HashableFunction(res_paths, closure=()))
 
+  if config.enable_key_reuse_checks.value:
+    # TODO(jakevdp)[key-reuse] make this import unconditional. This would require
+    #      JAX testing targets to have a dependency on jax:experimental-key-reuse.
+    try:
+      from jax.experimental import key_reuse  # pytype: disable=import-error
+    except ImportError:
+      pass
+    else:
+      key_reuse.check_jaxpr(jaxpr.jaxpr)
```

Comment:
Push this inside `_pjit_jaxpr`? Or look for where the other `check_jaxpr` is called.